docs/index.markdown

```diff
@@ -64,21 +65,19 @@ particular is just a big [reduction](https://developer.download.nvidia.com/asset
 although the reduction function requires some work to describe.
 
 These 2 algorithms themselves have sub-algorithms which you'll need to implement first.
-In total, the first stage has 4 challenges. After implementing these challenges you'll
-be well on your way to having a complete GPU implementation of the SNARK prover itself.
-
-### The stage 1 challenges
+In total, the first stage has 4 challenges. After implementing these algorithms you'll
+be well on your way to having a complete GPU implementation of the SNARK prover.
 
-The stage 1 challenges are
+### The Stage 1 challenges
 
-1. [Finite field arithmetic](/snark-challenge/problem-01-field-arithmetic.html). Reward of $50.
```

Comment:
suggestion to get rid of dollar amt here since it's nested in the challenge, and $25 or w/e is kind of underwhelming right next to the challenge link